flink-kubernetes/src/main/java/org/apache/flink/kubernetes/cli/KubernetesCliOptions.java

```diff
@@ -46,15 +44,16 @@ public static Option getOptionWithPrefix(Option option, String shortPrefix, Stri
 		.longOpt("clusterId")
 		.required(false)
 		.hasArg(true)
-		.desc(KubernetesConfigOptions.CLUSTER_ID.description().toString())
+		.desc("The cluster id that will be used for flink cluster. If it's not set, the client will generate " +
+			"a random UUID name.")
 		.build();
 
 	public static final Option IMAGE_OPTION = Option.builder("i")
 		.longOpt("image")
 		.required(false)
 		.hasArg(true)
 		.argName("image-name")
-		.desc(KubernetesConfigOptions.CONTAINER_IMAGE.description().toString())
+		.desc("Image to use for Flink containers")
```

Comment:
Also I am not sure why we make this change.

Reply:
`KubernetesConfigOptions.CONTAINER_IMAGE.description().toString()` will get `org.apache.flink.configuration.description.Description@166fa74d`, not the real description.

The cli option description may be different with config option. So i just put the real description text here.

Reply:
Thanks for the explanation, then it is better to put these reasons under the commit descriptions to let others know the motivation of this commit.